pkg/notes/model/repository.ts

```diff
@@ -38,3 +39,22 @@ export interface NoteAttachmentRepository {
   ): Promise<Result.Result<Error, void>>;
   findByNoteID(noteID: NoteID): Promise<Result.Result<Error, Medium[]>>;
 }
+
+export interface ReactionRepository {
+  create(
+    id: {
+      noteID: NoteID;
+      accountID: AccountID;
+    },
+    body: string,
+  ): Promise<Result.Result<Error, void>>;
+  findByID(id: {
+    noteID: NoteID;
+    accountID: AccountID;
+  }): Promise<Option.Option<Reaction>>;
+  findByAccountID(id: AccountID): Promise<Option.Option<Reaction[]>>;
```

Comment:
```suggestion
  reactionsByAccount(id: AccountID): Promise<Reaction[]>;
```
この `findByAccountID` メソッドは `NoteID` を指定しないようになっていますが, それで取得してくる `Reaction[]` が何を意味するのかメソッド名から読み取れないです. 「そのアカウントによって行われたすべてのリアクション」という意味であれば, `Option` で包む必要は無さそうです.

Reply:
修正しました．(689908c6005c4e543c193bed8a90c9f50201d86f)